akka-stream/src/main/scala/akka/stream/ContextMapStrategy.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream
+
+object ContextMapStrategy {
+
+  final case class Iterate[In, Ctx, Out](
+      iterateFn: (In, Ctx, Out, Int, Boolean) => Ctx,
+      emptyFn: Option[(In, Ctx) => (Out, Ctx)] = None)
+
+  /**
+   * Passthrough the same context [[Ctx] of the input element [[In]] to all output elements [[Out]].
```

Comment:
Do `[[` links work for type parameters? `Ctx` is missing one closing bracket.

Reply:
I'm not sure.  I'll check.